targets/targets.json

```diff
@@ -1726,13 +1730,15 @@
             "PORTIN",
             "PORTINOUT",
             "PORTOUT",
+            "RESET_REASON",
             "PWMOUT",
             "SERIAL",
             "SLEEP",
             "SPI",
             "SPISLAVE",
             "SPI_ASYNCH",
-            "STDIO_MESSAGES"
+            "STDIO_MESSAGES",
+            "WATCHDOG"
```

Comment:
WATCHDOG and RESET_REASON are added in the STM32 FAMILY level,
so it has to be removed from all STM32 targets after.
Thx